docs/index.md

```diff
@@ -260,7 +260,7 @@ And a payload like:
   "state": "<<client-supplied value, possibly empty>>",
   "registration":  {
     "id_token_signed_response_alg" : "ES256",
-    "id_token_encrypted_response_alg": "ECDH-ES",
+    "id_token_encrypted_response_kid": "<<encryption-key-id>>",
```

Comment:
If we eliminate `id_token_encrypted_response_alg` we may wan to add a note that the `kid` must refer to a key with alg=`ECDH-ES`

Reply:
I don't think we should add this field. It would be breaching the OIDC spec (and the DID SIOP spec). There's likely been conversation about this in the Dynamic Client Registration spec before, and that seems like a better place to pitch this change.